### PR TITLE
Documentation for Tasks, and Automation, Wrapping, Gradle & Maven Pipelines

### PR DESCRIPTION
**Pull request by jadecarino**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by jadecarino** [comment]
Hi, please can we get a review on the documentation? Aaron and I have documented the EventListeners, Tasks, and the Pipelines for Automation, Wrapping, Gradle and Maven. As we've discussed, the pipelines are pretty similar so we have tried to not be too repetitive by summarising what a pr-xxx and main-xxx build involve and then specific bits about the pipelines after. Would be helpful to get feedback on the technical explanations, formatting or just general proof reading. Thanks!
@hobbit1983 @techcobweb @CaroMac

File: README.md [hunk 1]
```diff
@@ -6,5 +6,303 @@ This repository is the single location for all the automation and CI/CD that occ
...
+
+# Pipelines
+
+
+## Event Listeners
+
+
+### github-pr-builder-listener
+
+This EventListener is triggered via webhook when a pull request is opened in a repository.
```
Comment (hobbit1983) [inline README.md:18]: which repositories in particular?

File: README.md [hunk 2]
```diff
@@ -6,5 +6,303 @@ This repository is the single location for all the automation and CI/CD that occ
...
+
+### Automation repository
+
+The build pipelines for the Automation repository are different to the build pipelines for the other repositories, as the Automation repository stores resources for the automation and CI/CD within Galasa, not source code. 
+
+### pr-automation
+
+This pipeline is triggered when a pull request is opened in the Automation repository.
+
+1. The pipeline starts by running the task 'git-verify'.
```
Comment (hobbit1983) [inline README.md:43]: what does that do?

**Review by hobbit1983 — APPROVED** [review]
Nice work - it's great to see all this documented

File: README.md (hunk 1 above)
Comment (jadecarino) [reply README.md:32]: I have added a list of which repositories have webhooks set up to trigger the EventListeners

**Comment by jadecarino** [comment]
Pull request 'synchronize' action not yet supported by PR build pipelines, so I have to merge in without running the PR build unfortunately.

**Pull request merged**